themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties

```
@@ -0,0 +1,171 @@
+# encoding: utf-8
+doSave=Uložit
+doCancel=Zrušit
+doLogOutAllSessions=Odhlásit všechny relace
+doRemove=Odstranit
+doAdd=Přidat
+doSignOut=Odhlásit se
+
+editAccountHtmlTitle=Upravit účet
+federatedIdentitiesHtmlTitle=Propojené identity
+accountLogHtmlTitle=Lod účtu
```

I'm sorry, but there is a typo. 
```suggestion
accountLogHtmlTitle=Log účtu
```

---

Oh, sorry for that :-/ ... fixed.